### PR TITLE
Updating standard report in pat_app dir

### PR DESCRIPTION
**Pull request by DavidGoldwasser**
Should work this way in 1.x or 2.x. In both cases charts will not be
visible if viewing offline. Also now enforce SI units for tabular
EnergyPlus results.

### CODE REVIEW

**Comment by nrel-bot-3** [comment]
@macumber @DavidGoldwasser @lgentile it has been 7 days since this pull request was last updated.

File: openstudiocore/src/pat_app/Measures/StandardReports/measure.rb [hunk 1]
```diff
@@ -148,49 +148,65 @@ def run(runner, user_arguments)
...
-        section = eval("OsLib_Reporting.#{method_name}(nil,nil,nil,true)")
-        section[:messages] = []
-        section[:messages] << "#{display_name} section failed and was skipped because: #{e}. Detail on error follows."
-        section[:messages] << ["#{e.backtrace.join("\n")}"]
-        @sections << section
 
       end
 
+    else
+      wrong_tabular_units_string = "IP units were provide, SI units were expected. Leave EnergyPlus tabular results in SI units to run this report."
```
Comment (macumber) [inline openstudiocore/src/pat_app/Measures/StandardReports/measure.rb]: should be "provided"

File: openstudiocore/src/pat_app/Measures/StandardReports/measure.rb [hunk 2]
```diff
@@ -148,49 +148,65 @@ def run(runner, user_arguments)
...
-          @sections << section
-          sections_made += 1
-          # look for emtpy tables and warn if skipped because returned empty
-          section[:tables].each do |table|
-            if not table
-              runner.registerWarning("A table in #{display_name} section returned false and was skipped.")
-              section[:messages] = ["One or more tables in #{display_name} section returned false and was skipped."]
+    # check units of tabular E+ results
+    column_units_query = "SELECT DISTINCT  units FROM tabulardatawithstrings WHERE ReportName='AnnualBuildingUtilityPerformanceSummary' and TableName='Building Area'"
+    energy_plus_area_units = sql_file.execAndReturnVectorOfString(column_units_query).get
```
Comment (macumber) [inline openstudiocore/src/pat_app/Measures/StandardReports/measure.rb]: calling .get and .first could fail, should have some error handling

File: openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb [hunk 3]
```diff
@@ -3228,7 +3227,7 @@ def self.zone_summary_section(model, sqlFile, runner, name_only = false)
     # data for query
     report_name = 'InputVerificationandResultsSummary'
     table_name = 'Zone Summary'
-    columns = ['', 'Area', 'Conditioned (Y/N)', 'Part of Total Floor Area (Y/N)', 'Volume', 'Multiplier', 'Gross Wall Area', 'Window Glass Area', 'Lighting', 'People', 'Plug and Process']
+    columns = ['', 'Area', 'Conditioned (Y/N)', 'Part of Total Floor Area (Y/N)', 'Volume', 'Multiplier', 'Above Ground Gross Wall Area', 'Underground Gross Wall Area', 'Window Glass Area', 'Lighting', 'People', 'Plug and Process']
```
Comment (macumber) [inline openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb]: This is a required 8.6 fix?

File: openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb [hunk 4]
```diff
@@ -2591,8 +2590,8 @@ def self.utility_bills_rates_section(model, sqlFile, runner, name_only = false)
     # create table
     energy_cost_table = {}
     energy_cost_table[:title] = 'Energy Cost Summary'
-    energy_cost_table[:header] = ['', 'Process Subtotal', 'Total Energy Cost']
-    energy_cost_table[:units] = ['', '$', '$']
+    energy_cost_table[:header] = ['', 'Total Energy Cost'] # skipping Process Subtotal
```
Comment (macumber) [inline openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb]: Is this an 8.6 issue or something else?

File: openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb (hunk 3 above)
Comment (DavidGoldwasser) [inline openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb]: Yes

File: openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb (hunk 4 above)
Comment (DavidGoldwasser) [inline openstudiocore/src/pat_app/Measures/StandardReports/resources/os_lib_reporting.rb]: This is not related to 8.6. @asparke2 just wanted to hide Process Subtotal which was confusing.

File: openstudiocore/src/pat_app/Measures/StandardReports/measure.rb (hunk 1 above)
Comment (DavidGoldwasser) [inline openstudiocore/src/pat_app/Measures/StandardReports/measure.rb]: fixing that now

File: openstudiocore/src/pat_app/Measures/StandardReports/measure.rb (hunk 2 above)
Comment (DavidGoldwasser) [inline openstudiocore/src/pat_app/Measures/StandardReports/measure.rb]: working on this now.

**Comment by DavidGoldwasser** [comment]
@macumber just made a commit to reflect your comments on the type and on .get and .first. Made these same changes in openstudio-measures repo copy of this measure.
